components/prism-asm6502.js

```diff
@@ -0,0 +1,28 @@
+Prism.languages.asm6502 = {
+	'comment': /;[^\n]*/,
+	'directive': {
+		pattern: /\.\w+ /,
```

Comment:
Is it intended to consume the final space here? If not, you might want to consider using a positive look-ahead assertion instead `(?= )`.